locale/pt_BR/admin.xml

```diff
@@ -93,8 +93,10 @@
 	<message key="admin.scheduledTask.startTime">Tarefa iniciada.</message>
 	<message key="admin.scheduledTask.stopTime">Tarefa parada.</message>
 	<message key="admin.scheduledTask.noLog">Tarefa não produziu registros (logs).</message>
-	<message key="admin.scheduledTask.openAccessNotification">Notificação de acesso aberto</message>
```

Comment:
You're right, I did it. Don't worry, it's already merged in. I will take a look why it's still being used. Thanks for your contribution. :)